jekyll/_cci2/api_v2_introduction.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: classic-docs
+title: "API v2 Introduction"
+short-title: "Introduction"
+description: "introduction to the CircleCI API v2"
+categories: [getting-started]
+order: 1
+---
+
+The CircleCI API v2 is a new API that may be used by internal and external users to make API calls to retrieve detailed information about workflows and pipelines.
+
+## Introduction to API v2
+
+CircleCI API v2 enables you to use a brand-new API with a set of endpoints and several new features that improve the API experience, in addition to optimizing how you use the API for your jobs. Some of these features include the ability to:
+
+- Retrieve a workflow by ID
+- Trigger pipelines with parameters
+- Trigger particular workflows within pipelines
+- Retrieve a pipeline by ID
+- Retrieve recent pipelines for an organization or for a project (ideally with branch filtering)
+
+## Getting Started with the API v2
+
+The v2 API is very similar to the current v1.1 API with some notable exceptions.
+
+Here is a simple example using `curl` to trigger a pipeline with parameters: 
+
+```
+curl -u ${CIRCLECI_TOKEN}: -X POST --header "Content-Type: application/json" -d '{
+  "parameters": {
+    "myparam": "./myspecialdir",
+    "myspecialversion": "4.8.2"
+  }
+}' https://circleci.com/api/v2/project/${project_slug}/pipeline
+```
+
+In the above example the `project_slug` would take the form :vcs/:org/:project. For example, the project slug `gh/CircleCI-Public/circleci-cli` tells `CircleCI` to use the project found in the GitHub organization CircleCI-Public in the repository named `circleci-cli`.
+
+### Authentication
+
+The CircleCI API v2 enables users to be authenticated by simply sending your API token as the username of the HTTP request. For example, if you have set `CIRCLECI_TOKEN` in your shell's environment, then you could then use `curl` with that token like the example shown below:
```

Comment:
Can we establish consistency on how we refer to the API? `v2 API` compared to `API v2`

Reply:
Also, consistency on how we present information. In the section where we explained `${project_slug}` we established examples and THEN explained the specifics, and here we are describing the specifics and then presenting examples.

Reply:
fixed